### ISSUE TITLE
Cannot read property 'value' of null

### ISSUE THREAD

**Issue by iHaiduk** [one-more/react-cron-builder#1]
When I click on clear button in select, I have next error:
```
Uncaught TypeError: Cannot read property 'value' of null
    at exports.getValue (preview.26a577d48087577ef808.bundle.js:3652)
    at Object.onChange (preview.26a577d48087577ef808.bundle.js:14520)
    at Object.setValue (preview.26a577d48087577ef808.bundle.js:13103)
    at Object.clearValue (preview.26a577d48087577ef808.bundle.js:13103)
    at Object.invokeGuardedCallback (preview.26a577d48087577ef808.bundle.js:5646)
    at executeDispatch (preview.26a577d48087577ef808.bundle.js:5646)
    at Object.executeDispatchesInOrder (preview.26a577d48087577ef808.bundle.js:5646)
    at executeDispatchesAndRelease (preview.26a577d48087577ef808.bundle.js:1763)
    at executeDispatchesAndReleaseTopLevel (preview.26a577d48087577ef808.bundle.js:1763)
    at Array.forEach (<anonymous>)
```

**Comment by one-more** [comment]
54abccf3a61d35674a9cf31e4d209cc41643d497

**Comment by one-more** [comment]
fixed 54abccf3a61d35674a9cf31e4d209cc41643d497
